docs/platform-engineers/debug/dry-run.md

```diff
@@ -1,3 +1,107 @@
 ---
 title: Dry Run
 ---
+
+Dry run will help you to understand what are the real resources which will to be expanded and deployed
+to the Kubernetes cluster. In other words, it will mock to run the same logic as KubeVela's controller 
+and output the results locally.
+
+For example, let's dry-run the following application:
+
+```yaml
+# app.yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: vela-app
+spec:
+  components:
+    - name: express-server
+      type: webservice
+      properties:
+        image: crccheck/hello-world
+        port: 8000
+      traits:
+        - type: ingress
+          properties:
+            domain: testsvc.example.com
+            http:
+              "/": 8000
+```
+
+```shell
+kubectl vela dry-run -f app.yaml
```

Comment:
Why don't we use `-dry-run` or `--dry-run` for the implementation of the CLI?

Reply:
I'm don't really know. It seems that in vela CLI there isn't any command beginning with a dash or double dash.